### PR TITLE
tiflash: add more explanation for parameter `bg_task_io_rate_limit`

### PR DESCRIPTION
**Pull request by lidezhu**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Joyinqin — APPROVED** [review]
/lgtm

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -56,12 +56,17 @@ delta_index_cache_size = 0
 
 ## 存储路径相关配置，从 v4.0.9 开始生效
 [storage]
-    ## [实验特性] 自 v5.0 引入，限制后台任务每秒写入的字节数。默认为 0，代表没有限制。目前为实验特性，不推荐在生产环境中使用。
+    ## [实验特性] 自 v5.0 引入，限制后台任务每秒写入的字节数。**目前为实验特性，不推荐在生产环境中使用。**
+    ## * 以 byte 为单位。目前不支持如 "10GB" 的设置。
+    ## * 默认为 0，代表没有限制。
+    ## 该参数主要针对 tiflash 部署在 aws ebs(gp2/gp3) 盘时的场景，用于控制后台任务对机器磁盘带宽的占用，提升 tiflash 查询性能的稳定性。
+    ## 在该场景下推荐配置范围 104857600 - 209715200 (即 100MB - 200MB)
```
Comment (flowbehappy) [inline tiflash/tiflash-configuration.md:63]:
```suggestion
    ## 在该场景下推荐配置为磁盘带宽的 50%
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -56,12 +56,17 @@ delta_index_cache_size = 0
 
 ## 存储路径相关配置，从 v4.0.9 开始生效
 [storage]
-    ## [实验特性] 自 v5.0 引入，限制后台任务每秒写入的字节数。默认为 0，代表没有限制。目前为实验特性，不推荐在生产环境中使用。
+    ## [实验特性] 自 v5.0 引入，限制后台任务每秒写入的字节数。**目前为实验特性，不推荐在生产环境中使用。**
+    ## * 以 byte 为单位。目前不支持如 "10GB" 的设置。
+    ## * 默认为 0，代表没有限制。
+    ## 该参数主要针对 tiflash 部署在 aws ebs(gp2/gp3) 盘时的场景，用于控制后台任务对机器磁盘带宽的占用，提升 tiflash 查询性能的稳定性。
+    ## 在该场景下推荐配置为磁盘带宽的 50%
```
Comment (flowbehappy) [inline tiflash/tiflash-configuration.md:63]:
```suggestion
    ## 该参数主要针对 tiflash 部署在 aws ebs(gp2/gp3) 盘时的场景，用于控制后台任务对机器磁盘带宽的占用，
    ## 提升 tiflash 查询性能的稳定性。在该场景下推荐配置为磁盘带宽的 50%。
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -56,12 +56,17 @@ delta_index_cache_size = 0
 
 ## 存储路径相关配置，从 v4.0.9 开始生效
 [storage]
-    ## [实验特性] 自 v5.0 引入，限制后台任务每秒写入的字节数。默认为 0，代表没有限制。目前为实验特性，不推荐在生产环境中使用。
+    ## [实验特性] 自 v5.0 引入，限制后台任务每秒写入的字节数。**目前为实验特性，不推荐在生产环境中使用。**
+    ## * 以 byte 为单位。目前不支持如 "10GB" 的设置。
+    ## * 默认为 0，代表没有限制。
+    ## 该参数主要针对 tiflash 部署在 aws ebs(gp2/gp3) 盘时的场景，用于控制后台任务对机器磁盘带宽的占用，
+    ## 提升 tiflash 查询性能的稳定性。在该场景下推荐配置为磁盘带宽的 50%。
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:63]:
```suggestion
    ## 该参数主要针对 TiFlash 部署在 AWS EBS (gp2/gp3) 盘时的场景，用于控制后台任务对机器磁盘带宽的占用。
    ## 提升 TiFlash 查询性能的稳定性。在该场景下推荐配置为磁盘带宽的 50%。
```

**Pull request merged**